ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -345,22 +350,32 @@ describe('ExpandedApp', () => {
 
   it('should allow launching RStudio when the RStudio app status is RUNNING', async () => {
     const appName = 'my-app';
+    const proxyUrl = 'https://example.com';
     const wrapper = await component(UIAppType.RSTUDIO, {
       appName,
       googleProject,
       status: AppStatus.RUNNING,
+      proxyUrls: {
+        rstudio: proxyUrl,
+      },
     });
 
+    const focusStub = jest.fn();
+    const windowOpenSpy = jest
+      .spyOn(window, 'open')
+      // @ts-ignore
```

Comment:
?

Reply:
Similar to the above but more extreme:

<img width="1632" alt="image" src="https://github.com/all-of-us/workbench/assets/31020403/416ef5f3-9476-4bfa-be5b-3a0591dd5b51">

I'm not too concerned about perfecting this particular test since [RW-9664](https://precisionmedicineinitiative.atlassian.net/browse/RW-9664) will remove the need for it entirely.

Reply:
This works but I wouldn't claim that it's better
```
.mockReturnValue({ focus: focusStub } as any as Window);
```

Reply:
I like that better since I think the intent is clearer. Updated.